iothub/device/src/DeviceClient.cs

```diff
@@ -394,14 +394,14 @@ public void SetRetryPolicy(IRetryPolicy retryPolicy)
 
 
         /// <summary>
-        /// Sends an event to device hub
+        /// Sends an event to device hub. Requires AMQP over WebSockets.
         /// </summary>
```

Comment:
This is generic one which works on all protocols. Batching is the one that works only on AMQP. Please revert this.